src/talos/moi_tests.py

```diff
@@ -101,10 +101,12 @@ def check_for_second_hit(
     if sample not in comp_hets:
         return []
 
-    partners = comp_hets[sample].get(first_variant, [])
-    if require_non_support:
-        return [partner for partner in partners if partner.sample_category_check(sample, allow_support=False)]
-    return partners
+    # thin out the possible partners by alt depth
+    return [
+        var
+        for var in comp_hets[sample].get(first_variant, [])
+        if not var.check_minimum_alt_depth(sample, min_alt_depth)
```

Comment:
I know it is not part of this PR, but the `check_minimum_alt_depth` method name is a bit curly (ie the `not` in front of it confused me)

Reply:
Yup, I think I should revisit all the method names to optimise the logic they're involved in...